api/src/main/proto/stellarstation/api/v1/monitoring/monitoring.proto

```diff
@@ -236,21 +236,36 @@ message PlanState {
   // The status of the plan.
```

Comment:
`// A status of a plan` and the field should have `// The status of the plan.`. It's pretty redundant but I think we should be extra-pedantic in our developer API.